Cargo.toml

```diff
@@ -13,6 +13,8 @@ license = "MIT OR Apache-2.0"
 repository = "https://github.com/model-checking/kani"
 documentation = "https://model-checking.github.io/kani/"
 homepage = "https://github.com/model-checking/kani"
+# N.B. Cargo.* is included automatically:
+include = ["/src", "/build.rs", "/rust-toolchain.toml", "/LICENSE-*", "/README.md"]
```

Comment:
Do we need to include files under `library/kani`, e.g. `gen_c_lib.c`, `kani_lib.c`?

Reply:
Nope those go in the release bundle. This is just the `kani-verifier` package on crates.io